msteams-platform/concepts/design/plan-responsive-tabs-for-teams-mobile.md

```diff
@@ -59,7 +59,7 @@ You can show the following information:
 * Data visuals, maps, and infographics.
 * Charts, graphs, and tables.
 
-:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-consumption.png" alt-text="Show the data in the form of visualization.":::
+:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-consumption.png" alt-text="The image shows the data visualization apps on Teams mobile platform .":::
```

Comment:
Not all data apps but an example of such an app. 
Remove the extra space from before the period.

Reply:
resolved